newIDE/app/src/MainFrame/Preferences/PreferencesProvider.js

```diff
@@ -239,11 +241,11 @@ export default class PreferencesProvider extends React.Component<Props, State> {
           },
         },
       }),
-      () => this._persistValuesToLocalStorage(this.state)
+      () => this._savePreferencesValues(this.state.values)
     );
   }
 
-  _loadValuesFromLocalStorage(): ?PreferencesValues {
+  _loadPreferencesValues(): ?PreferencesValues {
```

Comment:
Let's go back to the previous name - making clear this provider is tight to localStorage.